examples/osgi-test-example-bnd-workspace/org.osgi.test.example.player.impl/bnd.bnd

```diff
@@ -0,0 +1,7 @@
+# Build dependencies 
+-buildpath: \
+	osgi.annotation;version=latest,\
+	org.osgi.test.example.api;version=latest,\
+	org.osgi.service.component.annotations;version=latest
+
```

Comment:
I think you need  -testpath for the junit test that should be in this project. Where is the OpenBox test class?

Reply:
added `OpenBox.java` from mvn